src/App.js

```diff
@@ -1,8 +1,18 @@
 import './App.css';
+import DayCard from './components/dayCard/DayCard';
+import Wind from './assets/icons/wind.svg';
 
 function App() {
   return (
-    <div className="App" />
+    <div className='App'>
+      <DayCard
+        day='Saturday'
+        date='March 2nd'
+        temperature='36'
+        description='light rain'
+        imgSrc={Wind}
+      />
+    </div>
```

Comment:
I think the changes in this file should either not have been committed (as it is for testing purposes) or be in a separate PR (as it's implementing the `DayCard` and not part of the creation, which this PR is for)